observability/logging/logging-6.0/log6x-about.adoc

```diff
@@ -165,3 +181,154 @@ spec:
 ----
 
 . Verify that logs are visible in the Log section of the Observe tab in the OpenShift web console.
+
+=== Quick Start with OTEL
+
+To configure OTLP ingestion and enable the OTEL data model, follow these steps:
+
+:FeatureName: The OpenTelemetry Protocol (OTLP) output log forwarder
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+* Cluster administrator permissions
+
+.Procedure
+
+. Install the `OpenShift Logging` and `Loki` Operators from OperatorHub.
+
+. Create a `LokiStack` custom resource (CR) in the `openshift-logging` namespace:
++
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+  managementState: Managed
+  size: 1x.extra-small
+  storage:
+    schemas:
+    - effectiveDate: '2022-06-01'
+      version: v13
+    secret:
+      name: logging-loki-s3
+      type: s3
+    storageClassName: gp3-csi
+  tenants:
+    mode: openshift-logging
+----
+
+. Create a service account for the collector:
++
+[source,shell]
+----
+$ oc create sa collector -n openshift-logging
+----
+
+. Create a `ClusterRole` for the collector:
```

Comment:
This ClusterRole is created during installation of the Cluster Logging Operator and does not need to be created manually.

Reply:
Maybe keep this step as Optional and add a Note for the same. I believe it will help the user to understand the flow better (else we can delete this step also). Please let me know if you have other thoughts. Thanks!